test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/OnReferenceType/InterfaceTypeInOtherUsedOnlyBySavedAssembly.cs

```diff
@@ -4,6 +4,8 @@
 
 namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.OnReferenceType
 {
+	[IgnoreTestCase ("save action is not allowed")]
```

Comment:
Why not just delete the test case (or test that it fails)?

Reply:
Yep, it's gone now